canopeum_frontend/src/pages/Register.tsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable react/jsx-props-no-spreading -- Good practice for React Hook Form */
```

Comment:
Long-term I'd prefer reconfiguring this instead: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md

Let's keep the ignore for now, it'll help me better understand typical usages when I get to it.